spec-tests/src/test/java/org/candlepin/spec/ExportSpecTest.java

```diff
@@ -0,0 +1,649 @@
+import org.candlepin.dto.api.v1.RoleDTO;
+import org.candlepin.dto.api.v1.UserDTO;
+import org.candlepin.resource.CdnApi;
+import org.candlepin.resource.ConsumerTypeApi;
+import org.candlepin.resource.OwnerContentApi;
+import org.candlepin.resource.OwnerProductApi;
+import org.candlepin.resource.RolesApi;
+import org.candlepin.resource.RulesApi;
+import org.candlepin.resource.UsersApi;
+import org.candlepin.spec.bootstrap.assertions.OnlyInStandalone;
+import org.candlepin.spec.bootstrap.client.ApiClient;
+import org.candlepin.spec.bootstrap.client.ApiClients;
+import org.candlepin.spec.bootstrap.client.SpecTest;
+import org.candlepin.spec.bootstrap.client.api.ConsumerClient;
+import org.candlepin.spec.bootstrap.client.api.JobsClient;
+import org.candlepin.spec.bootstrap.client.api.OwnerClient;
+import org.candlepin.spec.bootstrap.client.request.Request;
+import org.candlepin.spec.bootstrap.client.request.Response;
+import org.candlepin.spec.bootstrap.data.builder.Branding;
+import org.candlepin.spec.bootstrap.data.builder.Cdns;
+import org.candlepin.spec.bootstrap.data.builder.ConsumerTypes;
+import org.candlepin.spec.bootstrap.data.builder.Consumers;
+import org.candlepin.spec.bootstrap.data.builder.Content;
+import org.candlepin.spec.bootstrap.data.builder.Owners;
+import org.candlepin.spec.bootstrap.data.builder.Pools;
+import org.candlepin.spec.bootstrap.data.builder.ProductAttributes;
+import org.candlepin.spec.bootstrap.data.builder.Products;
+import org.candlepin.spec.bootstrap.data.builder.Roles;
+import org.candlepin.spec.bootstrap.data.builder.Users;
+import org.candlepin.spec.bootstrap.data.util.ExportUtil;
+import org.candlepin.spec.bootstrap.data.util.UserUtil;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import org.apache.commons.codec.binary.Base64;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+@SpecTest
+class ExportSpecTest {
+    private static final String EXPORT_JOB_KEY = "ExportJob";
+    private static final String RECORD_CLEANER_JOB_KEY = "ImportRecordCleanerJob";
+    private static final String UNDO_IMPORTS_JOB_KEY = "UndoImportsJob";
+    /** The following are directory location paths within a manifest */
```

Comment:
This should use the one-line comment format so Javadoc and IDE doc parsers don't attach this line to the EXPORT_PATH declaration.

Reply:
Updated